authors/apps/core/tests/test_db_connection.py

```diff
@@ -0,0 +1,22 @@
+import logging
```

Comment:
This looks like an unused `import` ...

Reply:
@dmithamo1 the unused import has been noted and removed. Thanks for the feedback